src/pages/directory/[clubId].tsx

```diff
@@ -34,16 +34,14 @@ import {
   type InferGetStaticPropsType,
   type GetServerSidePropsContext,
 } from 'next';
-import DbProvider from '../../backend_tools/db_provider';
 import { createServerSideHelpers } from '@trpc/react-query/server';
 import { appRouter } from '@src/server/api/root';
 import { createInnerTRPCContext } from '@src/server/api/trpc';
 import { api } from '@src/utils/api';
+import { db } from '@src/server/db';
 
 export const getStaticPaths: GetStaticPaths = async () => {
-  const db = new DbProvider();
-  const clubs = await db.getAllClubs();
-
+  const clubs = await db.query.club.findMany();
```

Comment:
Should `{ with: { contacts: true } }` be added like it is in the trpc definition to get all clubs? Why is this condition here anyway? Are we not going to show clubs that have no contact info?

Reply:
I added it just so that it matched the previous firebase implementation. One thing that might be beneficial is moving the models away from the `models/` folder and trying to use `drizzle-zod` to generate the types based on the actual schema we create

Reply:
Yeah, that would remove a lot of the work we would have to do. Everything else looks good. I'll approve the pull request and you can merge when you're ready.